config/v3_6_experimental/translate/translate.go

```diff
@@ -16,10 +16,41 @@ package translate
 
 import (
 	"github.com/coreos/ignition/v2/config/translate"
+	"github.com/coreos/ignition/v2/config/util"
 	old_types "github.com/coreos/ignition/v2/config/v3_5/types"
 	"github.com/coreos/ignition/v2/config/v3_6_experimental/types"
 )
 
+func translateFileEmbedded1(old old_types.FileEmbedded1) (ret types.FileEmbedded1) {
```

Comment:
We should probably add that to 3.5.0 as well?

Reply:
I dont think so. You can think about these translations as stops on a train. 

3.0->3.1->3.3.->3.4->3.5->3.6-exp

When ignition receives work, it always takes the train from the input version to the destination. So if a change occurs at 3.3 which requires non standard mapping, that remap would be carried to 3.6-exp and ran. Thats why you dont see any duplication on the remaps between versions.

Reply:
OK, so that means that we only need to mask those bits when we read an older config that is internally represented as the latest? That feel weird now that I have written it. I though the translation was used to go from a newer to a lower version. Ah, so maybe one we've translated anything 3.6-exp and higher, we don't need to redo the translation again in lower versions.
So the translation would go: 3.6-exp->3.5->3.4->etc. ?

Reply:
Ah shoot, its entirely possible I have the direction backwards; Thankfully, we only need to translate once, regardless of the direction And yeah that would make sense.